src/schema/hello-schema.ts

```diff
@@ -1,17 +1,99 @@
 import { ApolloServer } from '@apollo/server';
 import { gql } from 'graphql-tag';
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
 
 // Define a schema using the GraphQL schema language
 const typeDefs = gql`
   type Query {
     hello: String
+    getUser(id: String!): User
+  }
+
+  type User {
+    id: String!
+    name: String!
+    email: String!
+    createdAt: String!
+    updatedAt: String!
```

Comment:
Acredito que não é obrigatório, mas poderia utilizar datetime do graphql scalars: https://the-guild.dev/graphql/scalars/docs/scalars/date-time